### DISCUSSION TITLE
How to use skupper with private repository

### DISCUSSION THREAD

**Discussion by qatrick** [skupperproject/skupper#1504] [Q&A] [answered]
How to use skupper with private registry? My env cant access internet so i need to push those skupper images to private registry. Thanks

**Comment by grs** [comment, marked as answer]
You can specify environment variables that will define the urls for the images in your private registry. The Env varvariable names for the different images are defined here: https://github.com/skupperproject/skupper/blob/main/pkg/images/image_utils.go

The env vars are used both by the CLI and by the site controller, depending on how you are configuring things.

**Reply by qatrick** [reply]
Thanks, it looks great
